templatestore/frontend/src/pages/templateScreen.js

```diff
@@ -361,6 +364,10 @@ class TemplateScreen extends Component {
     }
 
     render() {
+        if (Object.keys(this.state.config).length && !this.state.type && this.state.editable) {
```

Comment:
why this condition: `Object.keys(this.state.config).length && !this.state.type` ?